### ISSUE TITLE
User submitted tie sheets having issues.

### ISSUE THREAD

**Issue by mheagin** [agrc/PLSS#19]
Steve Woolsey
stevew@infowest.com
Submitted 4/17 and 1/18

Steve Woolsey was submitting pdf's from 5 to 9 MB for a single page. Told them to compress the photos they are embedding and try again. 
When they resubmitted them again, no photos so all were less than 1 MB now, but they still did not go through to the web page. 
I am able to download them from the email to the county for review.
We had another user submitted same day and they went through fine and a couple of days later more submitted and they worked too.
Questions.
Can you check the log and see what is going on with Steve Woolsey submittals?
Why is there a pdf associated with the email for the county review, but nothing is put into the ftp site, is it a size restriction for the ftp site?
Let me know if you have more questions
Thanks

**Comment by steveoh** [comment]
I added some extra logging around this part of the app to help figure out what is up. So next time he submits and it fails we can check out the logs.

**Comment by mheagin** [comment]
Update.......Talked to Jerrod , who works for Steve Woolsey today he is going to email the photos they are using. Then we can try to recreate the issue and see what is going on.

**Comment by steveoh** [comment]
Ok let me know when they do that and I will check the logs as well

**Comment by mheagin** [comment]
I have not heard back from the Woolsey team yet today. I am guessing they don't work over the weekend but might be in the office today given the fine spring weather.

**Comment by mheagin** [comment]
Talked to Steve W . they never used his seal when they submitted, he was going to into Settings and do that see if it works.  
While we talked he described how they did things and it sounds like he is submitting a tie sheet using our form and submitting their own doc as a existing sheet.  I went back and checked the ftp site and there are docs in there now but they are the submitted sheets.
I did the same thing today (5/4) and I got two docs with the ..(2) behind the second. but I also only got one email sent to the County even though I submitted two. 
I will have them submit some more and include the other sheet with the created document as an attachment which is an option.
We'll go from there.
